odk1-src/form-question-types.rst

```diff
@@ -1751,9 +1751,7 @@ type
 type
   :tc:`picker`
   
-When the :tc:`picker` appearance is added,
-the range widget is displayed 
-with a spinner-style select menu in a modal pop up.
+When the :tc:`picker` appearance is added, the range widget is displayed with a spinner-style select menu in a dialog. The value between horizontal lines is the selected value. Users can drag the spinner up and down or can tap on the value above to go up by one and on the value below to go down by one.
```

Comment:
s/drag/scroll

Reply:
Ah, I guess drag implies tap and hold. Ok, changed. Please squash when merging.